django/researchdata/views.py

```diff
@@ -10,11 +10,134 @@
                cwb_output_ngrams)
 
 
-class MonolingualCorporaInputView(TemplateView):
+class InputMonolingualView(TemplateView):
     """
-    Class-based view to show the Monolingual Corpora input template
+    Class-based view to show the input (monolingual) template
     """
-    template_name = 'researchdata/monolingual-corpora-input.html'
+    template_name = 'researchdata/input-monolingual.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['corpora_list'] = [
+            {
+                'id': 'uk',
+                'name': 'UK National Court'
+            },
+            {
+                'id': 'french',
+                'name': 'French National Court'
+            },
+            {
+                'id': 'austrian',
+                'name': 'Austrian National Court'
+            }
+        ]
+        return context
+
+
+class InputParallelView(TemplateView):
+    """
+    Class-based view to show the input (parallel) template
+    """
+    template_name = 'researchdata/input-parallel.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['corpora_list'] = [
```

Comment:
Ditto my comment above in InputMonolingualView - about a comment and a constant.

Reply:
Also, a personal style choice, I'd format this like:

```
blah = [
    {'id': 'birm_eng', 'name': 'English'},
    {...
```